### PR TITLE
Added information, how to increase connection limit

### PR DESCRIPTION
**Pull request by vincnetas**
## Overview

Added information, how to increase connection limit.

## Checklist

- [] Documentation is written and is accurate;
- [] `make check` passes with no errors
- [] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: src/maintenance/performance.rst [hunk 1]
```diff
@@ -212,6 +212,21 @@ This TCP buffering behaviour can be disabled via
 .. seealso::
     Bulk :ref:`load <api/db/all_docs>` and :ref:`store <api/db/bulk_docs>` API.
 
+Connection limit
+----------------
+
+`MochiWeb`_ is handling CouchDB requests.
+By default maximum number of connections is 2048. To change this limit, use
+server_options configuration variable. 'max' indicates maximum number of
```
Comment (nickva) [inline src/maintenance/performance.rst]: Let's use backticks \` for config string like say \`server_options\` or \`max\`

File: src/maintenance/performance.rst [hunk 2]
```diff
@@ -212,6 +212,21 @@ This TCP buffering behaviour can be disabled via
...
+connections.
+
+.. code-block:: ini
+
+    [httpd]
```
Comment (nickva) [inline src/maintenance/performance.rst]: For CouchDB 2.x

I think this needs to be `[chttpd]` to be applied to the main (clustered) port (by default 5984).

`[httpd]` would be for the backend (node local) port (by default 5986).


For CouchDB 1.x `[httpd]` works. But I think that should be a separate PR against the 1.6.x branch.

File: src/maintenance/performance.rst [hunk 3]
```diff
@@ -212,6 +212,21 @@ This TCP buffering behaviour can be disabled via
 .. seealso::
     Bulk :ref:`load <api/db/all_docs>` and :ref:`store <api/db/bulk_docs>` API.
 
+Connection limit
+----------------
+
+`MochiWeb`_ is handling CouchDB requests.
+By default maximum number of connections is 2048. To change this limit, use
```
Comment (flimzy) [inline src/maintenance/performance.rst]: This sentence is ungrammatical. I suggest either "The default maximum number of connections is..." or "By default, the maximum number of connections is..."

File: src/maintenance/performance.rst (hunk 1 above)
Comment (flimzy) [reply src/maintenance/performance.rst]: The sentence is also missing an article. It should read: "To change this limit, use the server_options configuration variable."

File: src/maintenance/performance.rst [hunk 4]
```diff
@@ -212,6 +212,21 @@ This TCP buffering behaviour can be disabled via
 .. seealso::
     Bulk :ref:`load <api/db/all_docs>` and :ref:`store <api/db/bulk_docs>` API.
 
+Connection limit
+----------------
+
+`MochiWeb`_ is handling CouchDB requests.
```
Comment (flimzy) [inline src/maintenance/performance.rst]: I suggest rewording to: "MochiWeb handles CouchDB requests."

The current wording is not incorrect, but is a bit awkward in native English.

**Comment by vincnetas** [comment]
Thank you for review. Fixed according to the feedback.

**Comment by flimzy** [comment]
+1 LGTM
